chef_master/source/custom_resources.rst

```diff
@@ -544,7 +544,7 @@ For example, a custom resource defines two properties (``content`` and ``path``)
 .. code-block:: ruby
 
    property :content, String
-   property :path, String, name_property: true
+   property :path, String, identity: true
```

Comment:
Is there a reason for this? Identity is pretty much legacy analytics stuff at this point.

Reply:
I changed the example to use name_property.